### DISCUSSION TITLE
在electerm终端里面自动开六个Tab

### DISCUSSION THREAD

**Discussion by gcqmkm02** [electerm/electerm#3517] [General]
大家好，
我不太清楚怎么用electerm -bo
我想自动开六个标签页。在两个用ssh连接服务器。以后在本地机器执行tmuxp等等。
这个怎么作呢？

**Comment by zxdong262** [comment]
设置里可以设置启动就打开的标签, 创建标签的时候可以设定执行的脚本，应该就可以了

**Reply by gcqmkm02** [reply]
找到了。谢谢。我在用.deb,好奇比snap有什么区别呢？需要执行electerm --no-sandbox。以后我还不太明白这个设置。执行man electerm不行。应该需要看哪个帮助文档呢？

**Reply by zxdong262** [reply]
功能没区别，没有文档，可以看看wiki： https://github.com/electerm/electerm/wiki
